qualification/antenna_channelised_voltage/test_channel_shape.py

```diff
@@ -119,7 +119,7 @@ async def sample(offsets: ArrayLike) -> np.ndarray:
         (np.arange(-0.5, 0.55, 0.1), -1.5, "Channel response (zoomed)"),
     ]:
         fig = Figure()
-        ax = fig.subplots()
+        ax = fig.add_subplot()
```

Comment:
What was wrong with the way it was?

Reply:
`subplots` can return a range of different things, and that was causing mypy to complain because some of them aren't usable the way `ax` is used. `add_subplot` has a more restricted return type.

Reply:
Right, makes sense. Matplotlib gives me the general feeling of being from an older time before type checking was even a thing, so trying to fit it on after the fact feels weird.